resource/closecaption_german.txt

```diff
@@ -461,89 +461,89 @@
 "[english]bastille.HoldingNorth0"		"<clr:205,127,50>Bastille: Covering north."
 "bastille.HoldingNorth1"		"<clr:205,127,50>Bastille: Ich decke die Nordseite ab."
 "[english]bastille.HoldingNorth1"		"<clr:205,127,50>Bastille: Covering northside."
-"bastille.HoldingNorth2"		"<clr:205,127,50>Bastille: Ich habe den Norden."
+"bastille.HoldingNorth2"		"<clr:205,127,50>Bastille: Halte den Norden."
 "[english]bastille.HoldingNorth2"		"<clr:205,127,50>Bastille: I've got the north."
-"bastille.HoldingNorth3"		"<clr:205,127,50>Bastille: Northside in Position."
+"bastille.HoldingNorth3"		"<clr:205,127,50>Bastille: Norden ist in Position."
 "[english]bastille.HoldingNorth3"		"<clr:205,127,50>Bastille: Northside in position."
 "bastille.HoldingSouth0"		"<clr:205,127,50>Bastille: Ich decke den Süden ab."
 "[english]bastille.HoldingSouth0"		"<clr:205,127,50>Bastille: Covering south."
-"bastille.HoldingSouth1"		"<clr:205,127,50>Bastille: Ich decke die Southside ab."
+"bastille.HoldingSouth1"		"<clr:205,127,50>Bastille: Ich decke die Südseite ab."
 "[english]bastille.HoldingSouth1"		"<clr:205,127,50>Bastille: Covering southside."
-"bastille.HoldingSouth2"		"<clr:205,127,50>Bastille: Ich habe den Süden."
+"bastille.HoldingSouth2"		"<clr:205,127,50>Bastille: Halte den Süden."
 "[english]bastille.HoldingSouth2"		"<clr:205,127,50>Bastille: I've got the south."
-"bastille.HoldingSouth3"		"<clr:205,127,50>Bastille: Southside in Position."
+"bastille.HoldingSouth3"		"<clr:205,127,50>Bastille: Süden ist in Position."
 "[english]bastille.HoldingSouth3"		"<clr:205,127,50>Bastille: Southside in position."
 "bastille.HoldingEast0"		"<clr:205,127,50>Bastille: Ich decke den Osten ab."
 "[english]bastille.HoldingEast0"		"<clr:205,127,50>Bastille: Covering east."
-"bastille.HoldingEast1"		"<clr:205,127,50>Bastille: Ich decke die Eastside ab."
+"bastille.HoldingEast1"		"<clr:205,127,50>Bastille: Ich decke die Ostseite ab."
 "[english]bastille.HoldingEast1"		"<clr:205,127,50>Bastille: Covering eastside."
-"bastille.HoldingEast2"		"<clr:205,127,50>Bastille: Ich habe den Osten."
+"bastille.HoldingEast2"		"<clr:205,127,50>Bastille: Halte den Osten."
 "[english]bastille.HoldingEast2"		"<clr:205,127,50>Bastille: I've got the east."
-"bastille.HoldingEast3"		"<clr:205,127,50>Bastille: Eastside in Position."
+"bastille.HoldingEast3"		"<clr:205,127,50>Bastille: Osten ist in Position."
```

Comment:
Ostseite in Position.
Usw. bei den nächsten Vorkommnissen.

Ich muss zugeben ich habe da selbst noch nicht so sehr darüber nachgedacht. Aber jetzt wo du es geändert hast machen wir es am besten noch genauer.